conda_smithy/templates/conda-forge.yml.ci-skel.tmpl

```diff
@@ -9,3 +9,4 @@ skip_render:
   - .github/CONTRIBUTING.md
   - .github/ISSUE_TEMPLATE.md
   - .github/PULL_REQUEST_TEMPLATE.md
+  - .github/workflows/webservices.yml
```

Comment:
Does the following work? I think it should.
```suggestion
  - .github/workflows
```

Reply:
But if we change the automerge options we want to remove that file so we cannot skip it on rerendering, right?